server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -308,7 +308,8 @@ export class AgenticChatController implements ChatHandlers {
                 chatResultStream
             )
         } catch (err) {
-            if (token?.isCancellationRequested) {
+            // TODO: On ToolValidationException, we want to show custom mynah-ui components making it clear it was cancelled.
+            if (token?.isCancellationRequested || err instanceof ToolApprovalException) {
```

Comment:
what UI would this eventually be?

Reply:
This is what it looks like on agentic-chat: 
<img width="1179" alt="image" src="https://github.com/user-attachments/assets/043e0c4b-27d3-4e83-af25-315db7a96c4a" />. 

Just some red text mostly.